std/datetime.d

```diff
@@ -28796,6 +28800,26 @@ private:
         initOnce!guard({tzset(); return true;}());
         return instance;
     }
+
+
+    // The Solaris version of struct tm has no tm_gmtoff field, so do it here
+    version(Solaris)
+    {
+        long tm_gmtoff(long stdTime) @trusted const nothrow
+        {
+            import core.stdc.time : localtime, gmtime, tm;
+
+            time_t unixTime = stdTimeToUnixTime(stdTime);
+            tm* buf = localtime(&unixTime);
+            tm timeInfo = *buf;
+            buf = gmtime(&unixTime);
+            tm timeInfoGmt = *buf;
+
+            return  (timeInfo.tm_sec-timeInfoGmt.tm_sec) +
+                    convert!("minutes", "seconds")(timeInfo.tm_min-timeInfoGmt.tm_min) +
+                    convert!("hours", "seconds")(timeInfo.tm_hour-timeInfoGmt.tm_hour);
```

Comment:
Please add spaces around the minus signs, and then I think that this is good to go.